apps/docs-snippets/src/guide/cookbook/transferring-assets.test.ts

```diff
@@ -79,7 +79,7 @@ describe(__filename, () => {
 
     // #region transferring-assets-2
     const transactionRequest = await sender.createTransfer(
-      destination.address,
+      destination.address.toB256(),
```

Comment:
Given we want to store enforcing `bech32` in the future, and we are using `fromDynamicInput` anyways, reckon it is better to start using `bech32` for address examples?

Reply:
I wonder if it's worth updating _all_ docs now or have a new issue for that. 🤔

Reply:
@danielbate I've changed a DOC test snippet by mistake, I wanted to add some cases for every one of the Account-affected methods just to validate that this is working.

@arboleya good question. I don't know if we need to add a complete page for it, but maybe place this info or a use-case in someplace. 

Also, this change will already be present on the Docs-generated API.